### DISCUSSION TITLE
Firebase Auth Token getting stored in local storage automatically

### DISCUSSION THREAD

**Discussion by hanishk** [firebase/flutterfire#10132] [Q&A]
Hi,
We are using firebase_auth for in our flutter app. I am not storing token in any local storage 
I am getting token using this code

Future<String>? get firebaseAuthToken =>
      _firebaseAuth.currentUser?.getIdToken(true);

But We gave our apk in testing to third party company they send token scrnsht by saying JWT token is saving in local storage.
I am not able to find the solution. Can anyone know this how to solve this?

Screenshot below:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/31143329/208104122-f388f9ed-6b26-4838-afb9-6cf2c2caacde.png">

**Comment by GubanovSergey** [comment]
@hanishk, as stated here https://firebase.google.com/docs/auth/web/auth-state-persistence, the athentication token is stored in local storage by default:

"If no user is signed in and no persistence is specified, the default setting will be applied (local in a browser app)."
